src/main/java/com/rocketden/main/dto/room/UpdateHostRequest.java

```diff
@@ -0,0 +1,12 @@
+package com.rocketden.main.dto.room;
+
+import com.rocketden.main.dto.user.UserDto;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+public class UpdateHostRequest {
+    private UserDto initiator;
+    private UserDto newHost;
```

Comment:
Can the host set themselves as the new host? I would assume yes, but want to make sure that if someone does that there aren't any unexpected consequences (bugs in the backend or whatnot for whatever reason).

Reply:
Just tested and yeah they can technically set themselves as the host (no errors occur, although I'm thinking in the frontend we can probably just prevent that from being an option)

Reply:
Ok awesome, thanks for verifying that, and yeah the frontend design sounds good.